OloEditor/SandboxProject/Assets/Scripts/LuaScripts/LuaPlayerController.lua

```
@@ -0,0 +1,193 @@
+-- LuaPlayerController.lua — Lua port of PlayerController.cs
+-- Top-down player with WASD movement, ability hotkeys (1/2/3), and camera zoom.
+-- Attach via LuaScriptComponent with ScriptFile = "Scripts/LuaScripts/LuaPlayerController.lua"
+
+local PlayerController = {}
+
+local moveSpeed = 5.0
+local myID = nil
```

Luacheck (1.2.0)

[warning] variable 'myID' is never accessed

(W231)